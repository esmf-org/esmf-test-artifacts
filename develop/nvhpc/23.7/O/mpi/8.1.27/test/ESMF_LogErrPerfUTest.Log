2025-05-10 03:59:20
20250510 042125.667 INFO             PET0 !!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!
20250510 042125.667 INFO             PET0 !!! THE ESMF_LOG IS SET TO OUTPUT ALL LOG MESSAGES !!!
20250510 042125.667 INFO             PET0 !!!     THIS MAY CAUSE SLOWDOWN IN PERFORMANCE     !!!
20250510 042125.667 INFO             PET0 !!! FOR PRODUCTION RUNS, USE:                      !!!
20250510 042125.667 INFO             PET0 !!!                   ESMF_LOGKIND_Multi_On_Error  !!!
20250510 042125.667 INFO             PET0 !!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!
20250510 042125.667 INFO             PET0 Running with ESMF Version   : v8.9.0b08-9-gea72ab2180
20250510 042125.667 INFO             PET0 ESMF library build date/time: "May 10 2025" "03:56:00"
20250510 042125.667 INFO             PET0 ESMF library build location : /glade/derecho/scratch/sacks/esmf-testing/nvhpc_23.7_mpi_O_develop/esmf
20250510 042125.667 INFO             PET0 ESMF_COMM                   : mpi
20250510 042125.667 INFO             PET0 ESMF_MOAB                   : enabled
20250510 042125.667 INFO             PET0 ESMF_LAPACK                 : enabled
20250510 042125.667 INFO             PET0 ESMF_NETCDF                 : enabled
20250510 042125.667 INFO             PET0 ESMF_PNETCDF                : disabled
20250510 042125.667 INFO             PET0 ESMF_PIO                    : enabled
20250510 042125.667 INFO             PET0 ESMF_YAMLCPP                : enabled
20250510 042125.667 INFO             PET0  Beginning Test, file ESMF_LogErrPerfUTest.F90, line 55
20250510 042125.667 INFO             PET0  NUMBER_OF_PROCESSORS            4
20250510 042125.667 INFO             PET0  perfLogFoundError:          1000  iterations took   8.3659999745577807E-006  seconds. =>    8.3659999745577808E-009  per iteration.
20250510 042125.667 INFO             PET0  PASS  Performance of ESMF_LogFoundError() 1000x Test, ESMF_LogErrPerfUTest.F90, line 64
20250510 042125.667 INFO             PET0  perfLogFoundError:         10000  iterations took   7.6516999797604512E-005  seconds. =>    7.6516999797604504E-009  per iteration.
20250510 042125.667 INFO             PET0  PASS  Performance of ESMF_LogFoundError() 10000x Test, ESMF_LogErrPerfUTest.F90, line 72
20250510 042125.668 INFO             PET0  perfLogFoundError:        100000  iterations took   7.6415199987422966E-004  seconds. =>    7.6415199987422974E-009  per iteration.
20250510 042125.668 INFO             PET0  PASS  Performance of ESMF_LogFoundError() 100000x Test, ESMF_LogErrPerfUTest.F90, line 80
20250510 042125.676 INFO             PET0  perfLogFoundError:       1000000  iterations took   7.7023030000873405E-003  seconds. =>    7.7023030000873411E-009  per iteration.
20250510 042125.676 INFO             PET0  PASS  Performance of ESMF_LogFoundError() 1000000x Test, ESMF_LogErrPerfUTest.F90, line 88
20250510 042125.676 INFO             PET0  PASS  Threshold check for ESMF_LogFoundError() 1000000x Test, ESMF_LogErrPerfUTest.F90, line 100
20250510 042125.676 INFO             PET0  Ending Test, file ESMF_LogErrPerfUTest.F90, line 107
20250510 042125.676 INFO             PET0  Finalizing ESMF
20250510 042125.666 INFO             PET1 !!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!
20250510 042125.667 INFO             PET1 !!! THE ESMF_LOG IS SET TO OUTPUT ALL LOG MESSAGES !!!
20250510 042125.667 INFO             PET1 !!!     THIS MAY CAUSE SLOWDOWN IN PERFORMANCE     !!!
20250510 042125.667 INFO             PET1 !!! FOR PRODUCTION RUNS, USE:                      !!!
20250510 042125.667 INFO             PET1 !!!                   ESMF_LOGKIND_Multi_On_Error  !!!
20250510 042125.667 INFO             PET1 !!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!
20250510 042125.667 INFO             PET1 Running with ESMF Version   : v8.9.0b08-9-gea72ab2180
20250510 042125.667 INFO             PET1 ESMF library build date/time: "May 10 2025" "03:56:00"
20250510 042125.667 INFO             PET1 ESMF library build location : /glade/derecho/scratch/sacks/esmf-testing/nvhpc_23.7_mpi_O_develop/esmf
20250510 042125.667 INFO             PET1 ESMF_COMM                   : mpi
20250510 042125.667 INFO             PET1 ESMF_MOAB                   : enabled
20250510 042125.667 INFO             PET1 ESMF_LAPACK                 : enabled
20250510 042125.667 INFO             PET1 ESMF_NETCDF                 : enabled
20250510 042125.667 INFO             PET1 ESMF_PNETCDF                : disabled
20250510 042125.667 INFO             PET1 ESMF_PIO                    : enabled
20250510 042125.667 INFO             PET1 ESMF_YAMLCPP                : enabled
20250510 042125.667 INFO             PET1  Beginning Test, file ESMF_LogErrPerfUTest.F90, line 55
20250510 042125.667 INFO             PET1  NUMBER_OF_PROCESSORS            4
20250510 042125.667 INFO             PET1  perfLogFoundError:          1000  iterations took   8.4060000062891049E-006  seconds. =>    8.4060000062891048E-009  per iteration.
20250510 042125.667 INFO             PET1  PASS  Performance of ESMF_LogFoundError() 1000x Test, ESMF_LogErrPerfUTest.F90, line 64
20250510 042125.667 INFO             PET1  perfLogFoundError:         10000  iterations took   7.6476999993246864E-005  seconds. =>    7.6476999993246868E-009  per iteration.
20250510 042125.668 INFO             PET1  PASS  Performance of ESMF_LogFoundError() 10000x Test, ESMF_LogErrPerfUTest.F90, line 72
20250510 042125.668 INFO             PET1  perfLogFoundError:        100000  iterations took   7.6415300009102793E-004  seconds. =>    7.6415300009102798E-009  per iteration.
20250510 042125.668 INFO             PET1  PASS  Performance of ESMF_LogFoundError() 100000x Test, ESMF_LogErrPerfUTest.F90, line 80
20250510 042125.676 INFO             PET1  perfLogFoundError:       1000000  iterations took   7.7045479999924282E-003  seconds. =>    7.7045479999924280E-009  per iteration.
20250510 042125.676 INFO             PET1  PASS  Performance of ESMF_LogFoundError() 1000000x Test, ESMF_LogErrPerfUTest.F90, line 88
20250510 042125.676 INFO             PET1  PASS  Threshold check for ESMF_LogFoundError() 1000000x Test, ESMF_LogErrPerfUTest.F90, line 100
20250510 042125.676 INFO             PET1  Ending Test, file ESMF_LogErrPerfUTest.F90, line 107
20250510 042125.676 INFO             PET1  Finalizing ESMF
20250510 042125.666 INFO             PET2 !!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!
20250510 042125.666 INFO             PET2 !!! THE ESMF_LOG IS SET TO OUTPUT ALL LOG MESSAGES !!!
20250510 042125.666 INFO             PET2 !!!     THIS MAY CAUSE SLOWDOWN IN PERFORMANCE     !!!
20250510 042125.666 INFO             PET2 !!! FOR PRODUCTION RUNS, USE:                      !!!
20250510 042125.666 INFO             PET2 !!!                   ESMF_LOGKIND_Multi_On_Error  !!!
20250510 042125.666 INFO             PET2 !!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!
20250510 042125.666 INFO             PET2 Running with ESMF Version   : v8.9.0b08-9-gea72ab2180
20250510 042125.666 INFO             PET2 ESMF library build date/time: "May 10 2025" "03:56:00"
20250510 042125.666 INFO             PET2 ESMF library build location : /glade/derecho/scratch/sacks/esmf-testing/nvhpc_23.7_mpi_O_develop/esmf
20250510 042125.666 INFO             PET2 ESMF_COMM                   : mpi
20250510 042125.667 INFO             PET2 ESMF_MOAB                   : enabled
20250510 042125.667 INFO             PET2 ESMF_LAPACK                 : enabled
20250510 042125.667 INFO             PET2 ESMF_NETCDF                 : enabled
20250510 042125.667 INFO             PET2 ESMF_PNETCDF                : disabled
20250510 042125.667 INFO             PET2 ESMF_PIO                    : enabled
20250510 042125.667 INFO             PET2 ESMF_YAMLCPP                : enabled
20250510 042125.667 INFO             PET2  Beginning Test, file ESMF_LogErrPerfUTest.F90, line 55
20250510 042125.667 INFO             PET2  NUMBER_OF_PROCESSORS            4
20250510 042125.667 INFO             PET2  perfLogFoundError:          1000  iterations took   8.2760000168491388E-006  seconds. =>    8.2760000168491388E-009  per iteration.
20250510 042125.667 INFO             PET2  PASS  Performance of ESMF_LogFoundError() 1000x Test, ESMF_LogErrPerfUTest.F90, line 64
20250510 042125.667 INFO             PET2  perfLogFoundError:         10000  iterations took   7.8330999940590118E-005  seconds. =>    7.8330999940590120E-009  per iteration.
20250510 042125.667 INFO             PET2  PASS  Performance of ESMF_LogFoundError() 10000x Test, ESMF_LogErrPerfUTest.F90, line 72
20250510 042125.668 INFO             PET2  perfLogFoundError:        100000  iterations took   7.6491499999065127E-004  seconds. =>    7.6491499999065131E-009  per iteration.
20250510 042125.668 INFO             PET2  PASS  Performance of ESMF_LogFoundError() 100000x Test, ESMF_LogErrPerfUTest.F90, line 80
20250510 042125.676 INFO             PET2  perfLogFoundError:       1000000  iterations took   7.7019029999974009E-003  seconds. =>    7.7019029999974016E-009  per iteration.
20250510 042125.676 INFO             PET2  PASS  Performance of ESMF_LogFoundError() 1000000x Test, ESMF_LogErrPerfUTest.F90, line 88
20250510 042125.676 INFO             PET2  PASS  Threshold check for ESMF_LogFoundError() 1000000x Test, ESMF_LogErrPerfUTest.F90, line 100
20250510 042125.676 INFO             PET2  Ending Test, file ESMF_LogErrPerfUTest.F90, line 107
20250510 042125.676 INFO             PET2  Finalizing ESMF
20250510 042125.666 INFO             PET3 !!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!
20250510 042125.666 INFO             PET3 !!! THE ESMF_LOG IS SET TO OUTPUT ALL LOG MESSAGES !!!
20250510 042125.666 INFO             PET3 !!!     THIS MAY CAUSE SLOWDOWN IN PERFORMANCE     !!!
20250510 042125.666 INFO             PET3 !!! FOR PRODUCTION RUNS, USE:                      !!!
20250510 042125.666 INFO             PET3 !!!                   ESMF_LOGKIND_Multi_On_Error  !!!
20250510 042125.666 INFO             PET3 !!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!
20250510 042125.666 INFO             PET3 Running with ESMF Version   : v8.9.0b08-9-gea72ab2180
20250510 042125.666 INFO             PET3 ESMF library build date/time: "May 10 2025" "03:56:00"
20250510 042125.666 INFO             PET3 ESMF library build location : /glade/derecho/scratch/sacks/esmf-testing/nvhpc_23.7_mpi_O_develop/esmf
20250510 042125.666 INFO             PET3 ESMF_COMM                   : mpi
20250510 042125.667 INFO             PET3 ESMF_MOAB                   : enabled
20250510 042125.667 INFO             PET3 ESMF_LAPACK                 : enabled
20250510 042125.667 INFO             PET3 ESMF_NETCDF                 : enabled
20250510 042125.667 INFO             PET3 ESMF_PNETCDF                : disabled
20250510 042125.667 INFO             PET3 ESMF_PIO                    : enabled
20250510 042125.667 INFO             PET3 ESMF_YAMLCPP                : enabled
20250510 042125.667 INFO             PET3  Beginning Test, file ESMF_LogErrPerfUTest.F90, line 55
20250510 042125.667 INFO             PET3  NUMBER_OF_PROCESSORS            4
20250510 042125.667 INFO             PET3  perfLogFoundError:          1000  iterations took   8.4460000380204292E-006  seconds. =>    8.4460000380204288E-009  per iteration.
20250510 042125.667 INFO             PET3  PASS  Performance of ESMF_LogFoundError() 1000x Test, ESMF_LogErrPerfUTest.F90, line 64
20250510 042125.667 INFO             PET3  perfLogFoundError:         10000  iterations took   7.7268999802981853E-005  seconds. =>    7.7268999802981856E-009  per iteration.
20250510 042125.667 INFO             PET3  PASS  Performance of ESMF_LogFoundError() 10000x Test, ESMF_LogErrPerfUTest.F90, line 72
20250510 042125.668 INFO             PET3  perfLogFoundError:        100000  iterations took   7.6416299998527393E-004  seconds. =>    7.6416299998527397E-009  per iteration.
20250510 042125.668 INFO             PET3  PASS  Performance of ESMF_LogFoundError() 100000x Test, ESMF_LogErrPerfUTest.F90, line 80
20250510 042125.676 INFO             PET3  perfLogFoundError:       1000000  iterations took   7.7048379998814198E-003  seconds. =>    7.7048379998814202E-009  per iteration.
20250510 042125.676 INFO             PET3  PASS  Performance of ESMF_LogFoundError() 1000000x Test, ESMF_LogErrPerfUTest.F90, line 88
20250510 042125.676 INFO             PET3  PASS  Threshold check for ESMF_LogFoundError() 1000000x Test, ESMF_LogErrPerfUTest.F90, line 100
20250510 042125.676 INFO             PET3  Ending Test, file ESMF_LogErrPerfUTest.F90, line 107
20250510 042125.676 INFO             PET3  Finalizing ESMF
