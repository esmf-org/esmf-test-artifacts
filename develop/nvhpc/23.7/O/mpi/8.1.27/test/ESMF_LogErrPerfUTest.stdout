2025-05-10 03:59:20
  Beginning Test, file ESMF_LogErrPerfUTest.F90, line 55
  NUMBER_OF_PROCESSORS            4
  PASS  Performance of ESMF_LogFoundError() 1000x Test, ESMF_LogErrPerfUTest.F90, line 64
  PASS  Performance of ESMF_LogFoundError() 10000x Test, ESMF_LogErrPerfUTest.F90, line 72
  PASS  Performance of ESMF_LogFoundError() 100000x Test, ESMF_LogErrPerfUTest.F90, line 80
  PASS  Performance of ESMF_LogFoundError() 1000000x Test, ESMF_LogErrPerfUTest.F90, line 88
  PASS  Threshold check for ESMF_LogFoundError() 1000000x Test, ESMF_LogErrPerfUTest.F90, line 100
  Ending Test, file ESMF_LogErrPerfUTest.F90, line 107
  PET            2  Test Elapsed Time     9.267091751098633       msec.
  Beginning Test, file ESMF_LogErrPerfUTest.F90, line 55
  NUMBER_OF_PROCESSORS            4
  PASS  Performance of ESMF_LogFoundError() 1000x Test, ESMF_LogErrPerfUTest.F90, line 64
  PASS  Performance of ESMF_LogFoundError() 10000x Test, ESMF_LogErrPerfUTest.F90, line 72
  PASS  Performance of ESMF_LogFoundError() 100000x Test, ESMF_LogErrPerfUTest.F90, line 80
  PASS  Performance of ESMF_LogFoundError() 1000000x Test, ESMF_LogErrPerfUTest.F90, line 88
  PASS  Threshold check for ESMF_LogFoundError() 1000000x Test, ESMF_LogErrPerfUTest.F90, line 100
  Ending Test, file ESMF_LogErrPerfUTest.F90, line 107
  PET            0  Test Elapsed Time     9.347200393676758       msec.
  Beginning Test, file ESMF_LogErrPerfUTest.F90, line 55
  NUMBER_OF_PROCESSORS            4
  PASS  Performance of ESMF_LogFoundError() 1000x Test, ESMF_LogErrPerfUTest.F90, line 64
  PASS  Performance of ESMF_LogFoundError() 10000x Test, ESMF_LogErrPerfUTest.F90, line 72
  PASS  Performance of ESMF_LogFoundError() 100000x Test, ESMF_LogErrPerfUTest.F90, line 80
  PASS  Performance of ESMF_LogFoundError() 1000000x Test, ESMF_LogErrPerfUTest.F90, line 88
  PASS  Threshold check for ESMF_LogFoundError() 1000000x Test, ESMF_LogErrPerfUTest.F90, line 100
  Ending Test, file ESMF_LogErrPerfUTest.F90, line 107
  PET            1  Test Elapsed Time     9.344100952148438       msec.
  Beginning Test, file ESMF_LogErrPerfUTest.F90, line 55
  NUMBER_OF_PROCESSORS            4
  PASS  Performance of ESMF_LogFoundError() 1000x Test, ESMF_LogErrPerfUTest.F90, line 64
  PASS  Performance of ESMF_LogFoundError() 10000x Test, ESMF_LogErrPerfUTest.F90, line 72
  PASS  Performance of ESMF_LogFoundError() 100000x Test, ESMF_LogErrPerfUTest.F90, line 80
  PASS  Performance of ESMF_LogFoundError() 1000000x Test, ESMF_LogErrPerfUTest.F90, line 88
  PASS  Threshold check for ESMF_LogFoundError() 1000000x Test, ESMF_LogErrPerfUTest.F90, line 100
  Ending Test, file ESMF_LogErrPerfUTest.F90, line 107
  PET            3  Test Elapsed Time     9.421110153198242       msec.
